src/test/starknet.t.sol

```diff
@@ -88,8 +89,17 @@ interface DaiLike {
 
 contract StarknetTests is DssSpellTestBase, ConfigStarknet {
 
+    event LogMessageToL2(
+            address indexed fromAddress,
+            uint256 indexed toAddress,
+            uint256 indexed selector,
+            uint256[] payload,
+            uint256 nonce,
+            uint256 fee
+        );
+
     function testStarknet() public {
-        if (l2Spell != 0) {
+        if (starknetValues.l2_spell != 0) {
             // Ensure the Pause Proxy has some ETH for the Starknet Spell
             assertGt(pauseProxy.balance, 0);
         }
```

Comment:
This part can be removed now as it is in the new test.

Reply:
Removed, but I'm not sure it's harmful to keep checks like this generally.